### ISSUE TITLE
Design Wayfinding: Floor vinyl and internal signage - Deadline Oct 11

### ISSUE THREAD

**Issue by kristinashu** [MozillaFoundation/Mozfest2016_production#89]
#### Info:

Based on epic ticket - Design Wayfinding #131 

Design a flexible floor vinyl system for wayfinding, includes name, floor number, and direction. 
### Need multiple vinyl signs for:
- [ ] Each Space (list of spaces https://github.com/MozillaFoundation/Mozfest2016_production/issues/88)
- [ ] "Main Stage" - Level 4
- [ ] "Debate and Discussion" stage - Walker Space - Level 0
- [ ] "Lunch" locations (multiple) 
- [ ] "MozHelp" desk (on 5th floor)
- [ ] Foyer floor branding

**Each sign will need Text + Arrow + Floor Level (as separate flexible pieces)**

We won't know where the Spaces will be until early Oct but can assume Floors 1-9 will have 1-2 Spaces each.

These locations will be updated in redpen: https://redpen.io/tp8c68c697ba47ced8

Locations we are not providing vinyl wayfinding for:
- Coffee
- Prayer room
- Creche
- Supplies room
- Toilets
- Water
- info booth
- Exhibits
- General branding (except for foyer floor)
#### Deadline:
- Oct 11
#### Specs:
- resolution: vector
- from last year
  Metamark Vinyl is good
  <img width="886" alt="48066ede-7efe-11e5-952f-910e2343d062" src="https://cloud.githubusercontent.com/assets/1068783/17943006/84c6dfbc-69ee-11e6-8f6b-188aeac7f4d1.png">
#### Last year's:
- Working files in Drive https://drive.google.com/open?id=0B2qu3ExKWIlvWjVHbVpvY0drUFE
- GitHub ticket https://github.com/mozilla/mozillafestival.org/issues/281

**Comment by kristinashu** [comment]
@edrushka @marcwalsh please add any details you can think of ^

**Comment by kristinashu** [comment]
Tracking locations as they are assigned here https://github.com/MozillaFoundation/Mozfest2016_production/issues/155

**Comment by kristinashu** [comment]
cc @ed-yau

**Comment by kristinashu** [comment]
@ed-yau will we be able to see the first draft of this on Monday? We would like to see all the pieces mentions at the top of this issue at their dimensions.

**Comment by ed-yau** [comment]
@kristinashu Yes I will have these ready by monday! Thanks for the notice!

**Comment by kristinashu** [comment]
Excellent, thanks!

**Comment by ed-yau** [comment]
@kristinashu @sabrinang @xmatthewx @Carrie-ann Please review updated / completed elements, and areas in progress for the way-finding in this redpen link. All these files are also currently shared on the google drive. [https://redpen.io/p/yfad874f695291a82c]

**Comment by kristinashu** [comment]
Design is done and opened a ticket for printing https://github.com/MozillaFoundation/Mozfest2016_production/issues/177

**Comment by kristinashu** [comment]
Vinyl was pre-cut and ready to go which worked out great. 

Brand vinyl for the foyer turned out well.
![artboard jpg](https://cloud.githubusercontent.com/assets/1068783/20194378/e1f9cdb2-a746-11e6-8008-80fff34e60eb.png)

We ended up only installing one set (Space name/level/arrow) per Space. They were installed on the the stairs just below the Space. It was good to have extra backups though, we taped all the vinyl in position before installing them but some were knocked off. 

<img width="421" alt="screen shot 2016-11-10 at 1 12 16 pm" src="https://cloud.githubusercontent.com/assets/1068783/20194473/5803d0de-a747-11e6-935c-82576c517a74.png">
